client/verta/tests/test_cli/test_registry.py

```diff
@@ -3,11 +3,10 @@
 import pytest
 from click.testing import CliRunner
 
+from verta import Client
 from verta._cli import cli
 
 
-pytest.skip("registry in Client not yet implemented", allow_module_level=True)
```

Comment:
Actually, we still need to skip these on master, because registry is not yet available on dev. Please use the skip() in master when you resolve the conflict (has an updated message)

Reply:
reverted